lib/myaccount/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './profileApi';
+export * from './emailApi';
+export * from './phoneApi';
```

Comment:
should also export types here?

Reply:
yes they are build there from the source file "types.ts". Don't you want `export * from './types'` in the index file?

Reply:
added, also added tsd to verify types